src/libsyntax/attr.rs

```diff
@@ -312,6 +312,15 @@ pub fn find_export_name_attr(diag: &SpanHandler, attrs: &[Attribute]) -> Option<
     })
 }
 
+pub fn contains_extern_indicator(attrs: &[Attribute]) -> bool {
+    contains_name(attrs, "no_mangle") ||
+        contains_name(attrs, "export_name") ||
```

Comment:
The only thing is that method takes a `SpanHandler` since it tries to actually parse the inner value. Should I add that to this method signature?